documentation/src/main/asciidoc/userguide/appendices/LegacyBasicTypeResolution.adoc

```diff
@@ -0,0 +1,346 @@
+:sourcedir: ../../../../test/java/org/hibernate/userguide/mapping
+:extrasdir: extras
+:originalextrasdir: ../chapters/domain/extras
+
+[[basic-legacy]]
+== Legacy resolution
```

Comment:
what about renaming the chapter something like `Legacy Type resolution` or `Legacy Basic Type resolution`